### PR TITLE
Add some link cleaning rules

### PR DESCRIPTION
**Pull request by alexklapheke**
`mailpgn.com`, `mailchimp.com`, `recruitics.com`, plus test URLs.

Tried to add a rule for `medium.com/m/global-identity` but it seems to lead to an infinite redirect loop.

### CODE REVIEW

File: background.js [hunk 1]
```diff
@@ -226,6 +226,22 @@ const urls_to_param_mappers = [
...
         urls: ["*://getpocket.com/redirect*", "*://www.getpocket.com/redirect*"]
+    },
+    {
+        urls: ["*://t.mailpgn.com/l/*"],
+        query_param: 'fl'
+    },
+    {
+        urls: ["*://*.mailchimp.com/mctx/clicks*"]
+    },
+    // Leads to infinite redirect loop
```
Comment (Riamse) [inline background.js:237]: I wonder if the infinite redirect loop is caused by Medium using, say, the `Referer` header - would be a good thing to investigate and if so, an easy feature to work on. Maybe another time!

File: test_urls.html [hunk 2]
```diff
@@ -90,6 +90,18 @@ <h1>Test page</h1>
...
     <a href="https://x.chip.de/linktrack/button/?url=https%3A%2F%2Fwww.vodafone.de%2Fnewsroom%2Fnetz%2Fstayhome-auf-jedem-handy-weil-wir-wollen-dass-ihr-sicher-seid%2F&c=8de2f3d865f23ac4473f53141fbbc468&type=text&articleId=126236983&referrer=https%3A%2F%2Fwww.chip.de%2Fnews%2FVodafone-mit-weiterer-Aktion-Was-bedeutet-StayHome_182567730.html&cms=ir&channel=News&bartId=1481&layoutId=4912&fqdn=www.chip.de">chip.de redirect</a>
 
+    <br /><br />
+    <a href="https://t.mailpgn.com/l/?u=f4285f76-f1be-4373-805b-f690af95ccd5&fl=https%3A%2F%2Fsiteoly.com">trypigeon.co redirect</a>
+
+    <br /><br />
+    <a href="https://us18.mailchimp.com/mctx/clicks?url=https%3A%2F%2Fmailchi.mp%2F5f701f9d65c0%2Fcongressman_bradley_byrne_staff-member_02-19-2020_mstownhall_130-230&h=5b02d3916fa64b22b6b286171fc98aa1b813d4f9dc17168518054717d42f001e&v=1&xid=4d8e63a7d8&uid=94352913&pool=&subject=">mailchimp redirect</a>
+
+    <br /><br />
+    <a href="https://medium.com/m/global-identity?redirectUrl=https://blog.unocoin.com/how-true-are-fears-of-a-cryptoasset-bubble-8dd119b15ccc">medium redirect</a>
```
Comment (Riamse) [inline test_urls.html:100]: Please add an explicit comment that Medium doesn't actually have a cleaner yet.

File: test_urls.md [hunk 3]
```diff
@@ -23,3 +23,11 @@ https://www.google.com/url?q=https://soundcloud.com&sa=D&ust=1504422217100000&us
...
+
+https://t.mailpgn.com/l/?u=f4285f76-f1be-4373-805b-f690af95ccd5&fl=https%3A%2F%2Fsiteoly.com
+
+https://us18.mailchimp.com/mctx/clicks?url=https%3A%2F%2Fmailchi.mp%2F5f701f9d65c0%2Fcongressman_bradley_byrne_staff-member_02-19-2020_mstownhall_130-230&h=5b02d3916fa64b22b6b286171fc98aa1b813d4f9dc17168518054717d42f001e&v=1&xid=4d8e63a7d8&uid=94352913&pool=&subject=
+
+https://medium.com/m/global-identity?redirectUrl=https://blog.unocoin.com/how-true-are-fears-of-a-cryptoasset-bubble-8dd119b15ccc
```
Comment (Riamse) [inline test_urls.md:31]: See above

**Review by Riamse — CHANGES_REQUESTED** [review]
Looks good!

File: test_urls.html (hunk 2 above)
Comment (alexklapheke) [reply test_urls.html]: OK, added to both files

**Review by Riamse — APPROVED** [review]

**Pull request merged**